ruby-challenges.rb

```diff
@@ -19,12 +56,46 @@
 # Expected output: ['Arizona', 'California', 'Idaho', 'Maine', 'Nevada', 'New Hampshire', 'Oregon', 'Rhode Island', 'Washington'] 
 
 # Pseudo code:
+#       input: a hash
+#       output: an array with hash values in alphabetical order
+#       process: make new hash?
+#                need to join all three arrays
+#                set in alpha order
+#                .sort
+#                google search points towards flatten method, which basically takes arrays with sub arrays joins them into one single array
```

Comment:
This is a good approach to build on, when you go to refactor think about how you could get the values of the hash.